component/authenticator/src/test/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticatorTest.java

```diff
@@ -177,46 +178,58 @@ public void testGetContextIdentifier() {
 
     @Test(description = "Test case for getFriendlyName() method.")
     public void testGetFriendlyName() {
+
         Assert.assertEquals(emailOTPAuthenticator.getFriendlyName(),
                 EmailOTPAuthenticatorConstants.AUTHENTICATOR_FRIENDLY_NAME);
     }
 
     @Test(description = "Test case for getAuthenticatorName() method.")
     public void testGetName() {
+
         Assert.assertEquals(emailOTPAuthenticator.getName(), EmailOTPAuthenticatorConstants.AUTHENTICATOR_NAME);
     }
 
     @Test(description = "Test case for retryAuthenticationEnabled() method.")
     public void testRetryAuthenticationEnabled() throws Exception {
+
         Assert.assertTrue(Whitebox.invokeMethod(emailOTPAuthenticator, "retryAuthenticationEnabled"));
     }
 
     @Test(description = "Test case for successful logout request.")
     public void testProcessLogoutRequest() throws Exception {
+
         when(context.isLogoutRequest()).thenReturn(true);
         doReturn(true).when(mockedEmailOTPAuthenticator).canHandle(httpServletRequest);
         AuthenticatorFlowStatus status = emailOTPAuthenticator.process(httpServletRequest, httpServletResponse,
                 context);
         Assert.assertEquals(status, AuthenticatorFlowStatus.SUCCESS_COMPLETED);
     }
 
-    @Test(description = "Test case for process() method when authenticated user is null.",
-            expectedExceptions = {AuthenticationFailedException.class})
-    public void testProcessWithoutAuthenticatedUser() throws AuthenticationFailedException, LogoutFailedException {
+    @Test(description = "Test case for process() method when authenticated user is null."
+//            , expectedExceptions = {AuthenticationFailedException.class}
```

Comment:
Why this is commented?

Reply:
Because that exception is no longer thrown due to the new implementation. 
"When authenticated user is null" case is handled not to throw an exception.

Reply:
Previously it was expected to throw an exception. But since it is handled, I have configured an assertion instead.